tools/c7n_gcp/c7n_gcp/resources/gke.py

```diff
@@ -140,6 +142,29 @@ def _get_location(cls, resource):
             return super()._get_location(cls.get_parent(resource))
 
 
+@KubernetesClusterNodePool.filter_registry.register('iam-policy')
+class KubernetesClusterNodePoolIamPolicyFilter(IamPolicyFilter):
+    permissions = ('resourcemanager.projects.getIamPolicy',)
+
```

Comment:
this could use a doc string example policy

Reply:
done